testsuites/lucene/src/main/resources/org/eclipse/rdf4j/sail/yeastract_raw.ttl

```diff
@@ -0,0 +1,1436 @@
+
+<urn:raw:yeastract#Yeast_id> <http://www.w3.org/2000/01/rdf-schema#label> "yeast_id" ;
```

Comment:
I will take that because I produced the testing dataset from third-party community.